pkg/owner/manager_test.go

```diff
@@ -293,13 +293,76 @@ func TestCluster(t *testing.T) {
 
 	logPrefix := fmt.Sprintf("[ddl] %s ownerManager %s", DDLOwnerKey, "useless id")
 	logCtx := logutil.WithKeyValue(context.Background(), "owner info", logPrefix)
-	_, err = owner.GetOwnerKey(context.Background(), logCtx, cliRW, DDLOwnerKey, "useless id")
+	_, _, err = owner.GetOwnerKeyInfo(context.Background(), logCtx, cliRW, DDLOwnerKey, "useless id")
 	require.Truef(t, terror.ErrorEqual(err, concurrency.ErrElectionNoLeader), "get owner info result don't match, err %v", err)
 	op, err := owner.GetOwnerOpValue(context.Background(), cliRW, DDLOwnerKey, logPrefix)
 	require.Truef(t, terror.ErrorEqual(err, concurrency.ErrElectionNoLeader), "get owner info result don't match, err %v", err)
 	require.Equal(t, op, owner.OpNone)
 }
 
+func TestWatchOwner(t *testing.T) {
```

Comment:
can you add a test that if DELETE happens between GetOwnerKeyInfo and watchOwner, it can retire normally

Reply:
> can you add a test that if DELETE happens between GetOwnerKeyInfo and watchOwner, it can retire normally

Done